course1/module4/Question4.js

```diff
@@ -0,0 +1,20 @@
+/*4.What happens if you pass a regular/invalid JSON string to JSON.parse? 
+What will happen if such an invalid function runs in the program? Will other parts of the code execute correctly after that?
+*/
+
+//for regular
+console.log(typeof JSON.parse('{"Harshal" : "sonawane"}'))
+
+//for invalid
+//console.log(typeof JSON.parse('{"Harshal" = "sonawane"}'))
```

Comment:
Please try all possible values for JSON.parse It can be undefine, null, empty strings, empty objects etc